src/decode.rs

```diff
@@ -297,7 +300,7 @@ pub fn decode_value<I: Iterator<Item = bool>>(ty: &Type, iter: &mut I) -> Result
 }
 
 /// Decode a 256-bit hash from bits.
-fn decode_hash<I: Iterator<Item = u8>>(iter: &mut BitIter<I>) -> Result<[u8; 32], Error> {
+pub fn decode_hash<I: Iterator<Item = u8>>(iter: &mut BitIter<I>) -> Result<[u8; 32], Error> {
```

Comment:
In c26c76b26623cab3f0d645be451331223a046a11:

It just occurred to me now that we could add a `fuzz` module which was gated on `#[cfg(fuzzing)]` which would expose this stuff. And then "only exposed for fuzzing" APIs could actually be exposed only for fuzzing.

Reply:
This is a really cool idea. I want to keep the API as small as possible, but fuzzers still need access.